### PR TITLE
scrcpy: update to 1.5.

### PR DESCRIPTION
**Pull request by look4ss**
Sorry for pulling this request again today, I made some unintended messes yesterday. So I am pulling this for the last time now.

### CODE REVIEW

File: srcpkgs/scrcpy/template [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # Template file for 'scrcpy'
 pkgname=scrcpy
-version=1.3
+version=1.5
 revision=1
 build_style=meson
-configure_args="-Db_lto=true
```
Comment (maxice8) [inline srcpkgs/scrcpy/template]: configure_args= is multiline, you need to keep it for the -Dprebuild_server option

**Comment by maxice8** [comment]
@SolitudeSF ok with the maintainership change ?

**Comment by SolitudeSF** [comment]
umm, yeah

**Comment by look4ss** [comment]
oh, sorry @SolitudeSF, I did not know about this maintainership change stuff, I thought it was ok to change the "maintainer" line in the template file when sending a pull request of some package update. Sorry.

Also, sorry to @maxice8 about all the mess that I made sending multiple pull requests of this commit yesterday, I am new to Void Linux package maintainership, I am still reading the documentation about this yet.

**Pull request closed without merging**